package main

func panicMain()  {
	//panic is the error handling

	panic("something back happened")
}